io.github.aginies.watermark.yml

```
@@ -0,0 +1,27 @@
+app-id: io.github.aginies.watermark
+runtime: 'org.freedesktop.Platform'
+runtime-version: '24.08'
+sdk: 'org.freedesktop.Sdk'
+command: watermark_app
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Notifications
+  - --share=network
+modules:
+  - python3-modules.json
+  - name: watermark_app
+    buildsystem: simple
+    build-commands:
+      - tar -xvf watermark-2.3.tar.gz
+      - install -Dm755 watermark-2.3/watermark_app.py /app/bin/watermark_app
+      - install -Dm644 watermark-2.3/io.github.aginies.watermark.png /app/share/icons/hicolor/256x256/apps/io.github.aginies.watermark.png
+      - install -Dm644 watermark-2.3/io.github.aginies.watermark.desktop /app/share/applications/io.github.aginies.watermark.desktop
+      - cp -a watermark-2.3/locale /app/share/
```

```suggestion
      - install -Dm755 watermark_app.py /app/bin/watermark_app
      - install -Dm644 io.github.aginies.watermark.png /app/share/icons/hicolor/256x256/apps/io.github.aginies.watermark.png
      - install -Dm644 io.github.aginies.watermark.desktop /app/share/applications/io.github.aginies.watermark.desktop
      - cp -a locale /app/share/
```

build will fail. It was more that just changing the type. But then you didn't check the build.